### PR TITLE
bill expiry api support for deeplinks

### LINKED ISSUE #12: Support Expire Bill API
Need to support the expire bill [API](https://docs.setu.co/payments/upi-deeplinks/quickstart#expire-bill--optional-)

### PR DESCRIPTION
**Pull request by dev1ender**
Bill expiry API added which expects Setu platform bill id as param

### CODE REVIEW

File: src/setu.js [hunk 1]
```diff
@@ -50,6 +50,15 @@ class Setu {
         }
     };
 
+    setBillExpiry(platformBillID=null) {
+        if (this.checkIfValuesExist()) {
```
Comment (nithinkashyapn) [inline src/setu.js]: verify is platformBillID exists as below

File: src/setu.js [hunk 2]
```diff
@@ -50,6 +50,15 @@ class Setu {
...
+            return requestHelper.post(
+                this.endpoints['bill-expiry']+platformBillID+'/expire',
```
Comment (nithinkashyapn) [inline src/setu.js]: can we have something like

this.endpoints['bill-expiry'] + `/${platformBillID}/expire'

File: src/setu.js (hunk 1 above)
Comment (dev1ender) [reply src/setu.js]: done

File: src/setu.js (hunk 2 above)
Comment (dev1ender) [reply src/setu.js]: done

File: src/helpers/endpoint.js [hunk 3]
```diff
@@ -12,7 +12,8 @@ module.exports = {
             reports: url + '/reports',
             'payment-link': url + '/payment-links',
             'mock-payment': url + '/triggers/funds/addCredit',
-            'retreive-reports': url + '/reports'
+            'retreive-reports': url + '/reports',
+            'bill-expiry':url + '/utilities/bills/',
```
Comment (nithinkashyapn) [inline src/helpers/endpoint.js]: remove the trailing slash as we are using / in setBillExpiry @dev1ender

Comment (dev1ender) [reply src/helpers/endpoint.js]: done

**Pull request merged**